src/components/primitive/chip-input/chip-input.component.tsx

```diff
@@ -64,7 +68,8 @@ export const ChipInput: React.FC<ChipInputComponentProps> = (props) => {
 
   return (
     <Container length={seedPhrasesLength()}>
```

Comment:
Property length does not signify the value that needs to be passed. Please add a meaningful name.